lib/verbalize/error.rb

```diff
@@ -1,3 +1,5 @@
 module Verbalize
   VerbalizeError = Class.new(StandardError)
+  # alias Verbalize::Error to Verbalize::VerbalizeError for now, for backwards compatibility
+  Error = Class.new(VerbalizeError)
```

Comment:
Do we care about backwards compatibility if this is already a major version changes?